<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="digit_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
font: 16pt &quot;Segoe UI&quot;;
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));
font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symb_plus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 0, 207);
font: 20pt &quot;Segoe UI&quot;;
color: rgb(41, 41, 41);
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symb_minus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 0, 207);
font: 20pt &quot;Segoe UI&quot;;
color: rgb(41, 41, 41);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symb_equal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 0, 207);
font: 20pt &quot;Segoe UI&quot;;
color: rgb(41, 41, 41);
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digit_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
font: 16pt &quot;Segoe UI&quot;;
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 41, 41);
color: qlineargradient(spread:pad, x1:1, y1:0.516227, x2:0, y2:0.226909, stop:0.352381 rgba(113, 0, 207, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symb_minus_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 0, 207);
font: 20pt &quot;Segoe UI&quot;;
color: rgb(41, 41, 41);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symb_minus_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 0, 207);
font: 20pt &quot;Segoe UI&quot;;
color: rgb(41, 41, 41);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symb_minus_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 0, 207);
font: 20pt &quot;Segoe UI&quot;;
color: rgb(41, 41, 41);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symb_minus_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 0, 207);
font: 20pt &quot;Segoe UI&quot;;
color: rgb(41, 41, 41);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="symb_minus_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 0, 207);
font: 20pt &quot;Segoe UI&quot;;
color: rgb(41, 41, 41);
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
